app/javascript/legacy/settings/index/branding/index.js

```diff
@@ -1,9 +1,7 @@
 // License: LGPL-3.0-or-later
 // npm
-const snabbdom = require('snabbdom')
```

Comment:
On all of these unused requires and imports, is there a way we can check for them as part of `eslint` or similar tool? That way we fix the problem for good.

Reply:
The rule for this is 'no-unused-vars' is already being used, however several files are in eslintignore.

Reply:
Let's do this a different way then: could you change the eslint configuration so it skips most of the rules on these old ignored files but does run the `no-unused-vars`? That should have a minimal effect on the git history but will help.

Reply:
Should a new issue be created for these changes? Because the purpose of the issue was to clean up code.

Reply:
It's solving the same problem so I'm fine using the same issue and PR in this case.

Reply:
Thanks for asking though!

Reply:
we managed to find a solution for this by putting a new eslintrc file inside the legacy folder. Would this be an acceptable solution? if yes this can also be replicated in react_legacy

Reply:
unfortunately we were not able to make an exception from the main rule only for legacy files, because apparently there is no way to deny file in the regex ref: https://github.com/selaux/eslint-plugin-filenames/issues/39. The solution described above solves the problem but it does not seem to be scalable, because we will always have to add new folders to the array, so this solution was not committed.

this is the root folder file :

![image](https://user-images.githubusercontent.com/31708472/155437671-ec23641f-a6d8-4397-980c-575f48555feb.png)

this is the legacy folder file:

![image](https://user-images.githubusercontent.com/31708472/155437774-691be8ad-9c42-4a5d-a618-06d9853b68d3.png)

Reply:
@wwahammy, do you know if this solution is acceptable? for us to make the commits

Reply:
@jpcirqueira I think so? I'm not sure I fully understand it, could you make the change and push it so I can see what it looks like?

Reply:
@wwahammy For this solution, @jpcirqueira and I added babel-eslint, to solve some parser errors. We also added overrides to legacy files to overwrite existing rules. There were 273 errors, do you think it makes sense to continue with this fix?

Reply:
@lucasA27 sorry about the delay in getting back to you, this slipped off my todo list. What was the reason that babel-eslint was needed?

Reply:
@wwahammy sorry for the delay, without babel-eslint some "parse errors" were happening in the legacy files.